code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -73,7 +73,7 @@
 
 /mob/living/simple_animal/mouse/Life()
 	..()
-	if(prob(0.5) && !ckey)
+	if(!stat && prob(0.5) && !ckey)

```

Comment:
this whole thing needs to be rewritten; it's gross and disgusting; we have a better resting/standing system to handle this; simple mobs can't even be unconscious, as far as I know.